app/indexers/descriptive_metadata_indexer.rb

```diff
@@ -24,7 +24,7 @@ def to_solr
       'contributor_orcids_ssim' => orcids,
       'sw_display_title_tesim' => title,
       'sw_subject_temporal_ssim' => subject_temporal,
-      'sw_subject_geographic_ssim' => subject_geographic,
+      'sw_subject_geographic_ssim' => stanford_mods_record.geographic_facet.uniq,
```

Comment:
It looks like all the test changes below are negative tests now. What *does* the change now map to?